.vscode/settings.json

```diff
@@ -1,4 +1,10 @@
 {
+    "editor.rulers": [88],
```

Comment:
Since Black automatically takes care of wrapping lines, I think we should just disable E501 and not force a specific ruler size.

Reply:
I agree, and have just removed `editor.rulers`.